tests/benchmarks/multicore/idle_clock_control/src/main.c

```diff
@@ -37,6 +37,29 @@ const struct nrf_clock_spec test_clk_specs_hsfll[] = {
 	},
 };
 
+const struct nrf_clock_spec test_clk_specs_global_hsfll[] = {
+	{
+		.frequency = MHZ(320),
+	},
+	{
+		.frequency = MHZ(256),
+	},
+	{
+		.frequency = MHZ(128),
+	},
+	{
+		.frequency = MHZ(64),
+	},
+};
+
+static const struct test_clk_ctx global_hsfll_test_clk_ctx[] = {
+	{
+		.clk_dev = DEVICE_DT_GET(DT_NODELABEL(hsfll120)),
+		.clk_specs = test_clk_specs_global_hsfll,
+		.clk_specs_size = ARRAY_SIZE(test_clk_specs_global_hsfll),
+	},
```

Comment:
this looks like it's reverting back to the old way of doing it from the newer way of having it done by Kconfig... what am I missing?

Reply:
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_spim_loopback/Kconfig#L48 but can see here it's not just checking one speed so ok